spring-integration-core/src/test/java/org/springframework/integration/endpoint/PseudoTransactionalMessageSourceTests.java

```diff
@@ -402,6 +408,7 @@ protected void doPoll(SourcePollingChannelAdapter adapter) {
 			method.invoke(adapter);
 		}
 		catch (Exception e) {
+			e.printStackTrace();
```

Comment:
No `e.printStackTrace()`, please, at all.
It doesn't provide much info and just pollutes logs on CI builds